articles/architecture-scenarios/b2c/user-logout.md

```diff
@@ -0,0 +1,68 @@
+---
+title: User Logout
+description: What to consider for user logout
+toc: true
+topics:
+    - b2c
+    - ciam
+    - logout
+    - sessions
+contentType: concept
+useCase:
+    - user-logout
+---
+# User Logout
+
+[Logout](/logout) is the act of terminating an authenticated session when they're no longer needed, minimizing the likelihood that unauthorized parties can "take over" the session. This is achieved by provisioning a logout option on the user interface you provide to your users.
+
+Multiple types of sessions can be created when a user logs in (e.g., local application sessions, Auth0 session, third-party Identity Provider sessions), and you will need to determine which of these sessions need to be terminated when the user clicks **Logout**.
+
```

Comment:
We seem to have lost the best practice panel from the original B2C Welcome

Reply:
Since we don't actually have a best practices UI element in the docs site, we're just using note boxes, and in turn my guess is that everyone is editing as we normally would, and trying to avoid unnecessary callout boxes. If it's mandatory that we have one in each section, I wonder if maybe we should make it very explicit, like

```
::: panel Best Practice: Foo Bar
Some Interesting Stuff
:::
```

WDYT @tamigoodall ? Or is that too much

Reply:
Thanks @jeffreylees :-) 

So some other sections seem to be using `::: panel Best Practice` - check out the User Provisioning section @ https://github.com/auth0/docs/pull/7310/files#diff-5d3b40e71f119d8edf0966fae8b3c1fc - whilst I've also seen `:::panel TL;DR` in others (https://github.com/auth0/docs/pull/7310/files#diff-ffadf9374e44aa30685e5938b5f01efe).

The idea is that one could read just the Best Practices - so very much a TL;DR - if one didn't want, or didn't have time to, dig through the rest of the text :-) So we really would like to keep *all* of the panels we put in the original B2C Welcome.

Reply:
And I think that whatever we do we should at least be consistent within the same body of work.